mojaloop/values.yaml

```diff
@@ -8054,7 +11326,20 @@ mojaloop-bulk:
 
         ## MongoDB Configuration for Object Store
         objstore_disabled: false
-        objstore_uri: *OBJSTORE_URI
+        mongo_host: *OBJSTORE_MONGO_HOST
+        mongo_port: *OBJSTORE_MONGO_PORT
+        mongo_user: *OBJSTORE_MONGO_USER
+        ## Secret-Management
+        ### Set this if you are using a clear password configured in the config section
+        mongo_password: *OBJSTORE_MONGO_PASSWORD
```

Comment:
```suggestion
        mongo_password: ''
```

Reply:
@elnyry-sam-k this is arelady set on the top-level config --> https://github.com/mojaloop/helm/blob/feat/%233082-utilise-externalised-secrets-for-user-password-management/mojaloop/values.yaml#L2